package.json

```diff
@@ -9,10 +9,10 @@
     "webpack": "./node_modules/webpack/webpack.js"
   },
   "engines": {
-    "node": "^10.x"
+    "node": "^18.x"
   },
   "scripts": {
-    "build": "yarn && yarn lint && yarn test && NODE_ENV=production webpack -p --bail --progress",
+    "build": "yarn && yarn test && rm -rf client/dist/* && NODE_ENV=production webpack --mode production --bail --progress",
```

Comment:
The `-p` option alias has been removed. `--mode production` must now be used explicitly